wasmbinding/export_test.go

```diff
@@ -2,6 +2,10 @@ package wasmbinding
 
 import "github.com/cosmos/cosmos-sdk/codec"
 
-func SetWhitelistedQuery(queryPath string, protoType codec.ProtoMarshaler) {
+func SetWhitelistedQuery[T any, PT protoTypeG[T]](queryPath string, protoType PT) {
 	setWhitelistedQuery(queryPath, protoType)
 }
+
+func GetWhitelistedQuery(queryPath string) (codec.ProtoMarshaler, error) {
```

Comment:
is this needed anywhere? we should not expose it

Reply:
Its is only exported here for the test suite, it is still private everywhere else.

Reply:
ah ok sorry I did not see!